example/moving_average.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Jiri Nytra (jiri.nytra at gmail dot com)
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <flux.hpp>
+
+#include <cstddef>
+#include <cassert>
+#include <deque>
+#include <vector>
+
+struct sliding_window_t
+{
+    explicit sliding_window_t(std::size_t N): size(N) {
+    }
```

Comment:
This is very nit-picky, but uppercase `N` here makes it look like a template parameter: I normally use lowercase names for runtime variables. Also for a constructor that can fit on a single line, I'd usually format it like this:

```suggestion
    explicit sliding_window_t(std::size_t n) : size(n) {}
```

Reply:
Thanks, fixed.